js/player.js

```diff
@@ -14,15 +14,21 @@ import * as FrameURL from "./player/FrameURL";
 
 window.addEventListener("load", function () {
 
-    Presentation.init(document.querySelector("svg"));
-    Viewport.init(Presentation).onLoad();
+    var presentation = Presentation.init(document.querySelector("svg"));
+    var viewport = Viewport.init(Presentation).onLoad();
```

Comment:
I did this on purpose because of readability, but I really don't care that much. :+1: